src/fetch_response.js

```diff
@@ -72,4 +76,14 @@ export class FetchResponse {
       return Promise.reject(new Error(`Expected a Turbo Stream response but got "${this.contentType}" instead`))
     }
   }
+
+  async activeScript () {
+    if (this.isScript) {
+      const script = document.createElement('script')
```

Comment:
UJS also sets a `nonce` for stricter CSP policies.

https://github.com/rails/rails/blob/7f80288ee5ff95a37232f85a0e1fed823cf0141b/actionview/app/assets/javascripts/rails-ujs.esm.js#L31-L34

```suggestion
      const script = document.createElement('script')
      const metaTag = document.querySelector("meta[name=csp-nonce]");
      const nonce = metaTag && metaTag.content;
      if (nonce) { script.setAttribute("nonce", nonce) }
```